src/typescript/11-keyof-typeof/keyof-typeof.md

```diff
@@ -28,7 +31,7 @@ type Dino = {
 If we were to create a new type called `Dino` using `keyof` we would see `"name" | "type" | "age"`
 
 ```ts
-type DinoKeys = keyof Dino; // "name" | "type" | "age"
+type DinoKeys = keyof Dino;
```

Comment:
I would leave the comment or put it on the next line so it’s clear what `DinoKeys` is.